### PR TITLE
updated readme.md

### PR DESCRIPTION
**Pull request by wendtbw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,11 +100,11 @@ make deploy-schema
...
-To run a local proxy that connect to your deployed pod run the following:
+To run a local proxy that connects to your deployed pod run the following:
 ```bash
 make proxy
 ```
 
 ## Contributing Code, Reporting Bugs and Suggesting Features
 
-As an organization committed to extreme transparency, collaboration, and open-sourcing all of our work, we welcome participation from anyone willing to devote some time and energy to help shape DAV - whether you are a first time contributor, a veteran open-sourcerer, or just looking to suggest some ideas.
+As an organization committed to extreme transparency, collaboration, and open-sourcing in all of our work, we welcome participation from anyone willing to devote some time and energy to help shape DAV - whether you are a first time contributor, a veteran open-sourcerer, or just looking to suggest some ideas.
```
Comment (TalAter) [inline README.md]: I prefer the original here.
We are committed to open-sourcing all of our work.

Comment (wendtbw) [reply README.md]: No worries.  My first pull request so I was excited to help!!  Regards.

Comment (TalAter) [reply README.md]: That's great to hear! I hope it's the first of many 👍 

Can you add another commit removing the "in " you added to line 110?

**Comment by TalAter** [comment]
Awesome 🎆 

Thank you @wendtbw

**Pull request merged**